expectations-of-foundation.md

```diff
@@ -0,0 +1,20 @@
+# Expectations
+With the intent to merge the Node.js Foundation and JS Foundation being announced, those interested in the continued support of Node.js and JS-related projects are working through expectations and structure for what the potential new foundation could look like. 
+
+This document is for the initial phase of work for the Bootstrap team to draft requirements that overlap with scope of the Board in order to provide an outline and structure for the foundation attuned to the needs of the projects and ecosystem stakeholders who have participated.
+
+- Communicated scope for the foundation and board
+- Annual reviews of staff, NPS, board, programs, support services to projects for accountability and incremental improvement
+- The foundation staff, executive director, and board should follow a leadership style that prioritizes and promotes the well-being of the project and those who prioritize and promote a healthy project and ecosystem.
+- Project representation on the Board
+  - Community Representation should sit on the Board directly. Projects could share this representation.
+  - Community representation on the Board: 3 seats guaranteed for community representation
+- Board must establish Cross Project Council focused on the specific needs of the member projects
+  - Allow projects to self-organize amongst themselves. Processes are defined and driven by project e.g. CPC is not structured by the Board
+- Non-corporate member representation in budget/finance subcommittee
+- Projects values should drive foundation focus and priority
+- No "pay to play" in technical participation
+- Earmarking allowed with flexibility in budgeting for concerns and needs of individual projects
+- Value satisfaction: regular feedback from projects
+- Explicit acknowledgement and documention for the ability of projects to leave the foundation
+- Foundation should document a clear process for trademark usage to enable community not-for-profit efforts
```

Comment:
I think this could be expanded. Node.js TSC and CommComm cannot use the trademark directly. It’s not only not-for-profit efforts, but even project representation.

Reply:
Has this been a problem in the past? I wasn't aware, if so, which I'd love to hear the details of if there's a case of it.

If folks aren't changing the trademark and are using it -for- the Node.js project, we should already be able to use it. Thus why it's good to have documentation around it. Modification is not okay without permission, and you can't make money off of it.

Reply:
I consider a problem that the TSC or CommComm cannot publish a message on social media as nodejs, as an example.

If somebody has access, this is clearly not documented anywhere.

Reply:
That is a good point, we can ask Zibby, but you need to know that you can ask her.

Reply:
FWIW on the JSF side the projects manage and run their own social channels, and would expect to continue doing so. Is the intent that the board and say a "brand/marketing committee" will be responsible for the new _@$Foundation's_ social account and Node.js (and other projects) are responsible for their own accounts? so the expectation is that the node foundation would turn over the use of @nodejs to the project?